workspace/assets/css/lib/class-z-index.less

```diff
@@ -1,76 +1,76 @@
+.z-index-199          {z-index:      199;}
+.z-index-200          {z-index:      200;}
+.z-index-201          {z-index:      201;}
 
-.z-index-299         {z-index:      299;}
-.z-index-300         {z-index:      300;}
-.z-index-301         {z-index:      301;}
+.z-index-299          {z-index:      299;}
+.z-index-300          {z-index:      300;}
+.z-index-301          {z-index:      301;}
 
-.z-index-399         {z-index:      399;}
-.z-index-400         {z-index:      400;}
-.z-index-401         {z-index:      401;}
+.z-index-399          {z-index:      399;}
+.z-index-400          {z-index:      400;}
+.z-index-401          {z-index:      401;}
 
-.z-index-499         {z-index:      499;}
-.z-index-500         {z-index:      500;}
-.z-index-501         {z-index:      501;}
+.z-index-499          {z-index:      499;}
+.z-index-500          {z-index:      500;}
+.z-index-501          {z-index:      501;}
 
-.z-index-1000        {z-index:     1000;}
-.z-index-2000        {z-index:     2000;}
-.z-index-3000        {z-index:     3000;}
-.z-index-4000        {z-index:     4000;}
-.z-index-5000        {z-index:     5000;}
-.z-index-6000        {z-index:     6000;}
-.z-index-7000        {z-index:     7000;}
-.z-index-8000        {z-index:     8000;}
-.z-index-9000        {z-index:     9000;}
-.z-index-10000       {z-index:    10000;}
+.z-index-1000         {z-index:     1000;}
+.z-index-2000         {z-index:     2000;}
+.z-index-3000         {z-index:     3000;}
+.z-index-4000         {z-index:     4000;}
+.z-index-5000         {z-index:     5000;}
+.z-index-6000         {z-index:     6000;}
+.z-index-7000         {z-index:     7000;}
+.z-index-8000         {z-index:     8000;}
+.z-index-9000         {z-index:     9000;}
+.z-index-10000        {z-index:    10000;}
 
-.z-index-100000      {z-index:   100000;}
-.z-index-500000      {z-index:   500000;}
-.z-index-1000000     {z-index:  1000000;}
+.z-index-100000       {z-index:   100000;}
+.z-index-500000       {z-index:   500000;}
+.z-index-1000000      {z-index:  1000000;}
 
-.z-index-max-minus-9 {z-index:  9999991;}
-.z-index-max-minus-8 {z-index:  9999992;}
-.z-index-max-minus-7 {z-index:  9999993;}
-.z-index-max-minus-6 {z-index:  9999994;}
-.z-index-max-minus-5 {z-index:  9999995;}
-.z-index-max-minus-4 {z-index:  9999996;}
-.z-index-max-minus-3 {z-index:  9999997;}
-.z-index-max-minus-2 {z-index:  9999998;}
-.z-index-max-minus-1 {z-index:  9999999;}
-.z-index-max         {z-index: 10000000;}
\ No newline at end of file
+.z-index-max-minus-10 {z-index:  9999991;}
+.z-index-max-minus-9  {z-index:  9999991;}
+.z-index-max-minus-8  {z-index:  9999992;}
+.z-index-max-minus-7  {z-index:  9999993;}
+.z-index-max-minus-6  {z-index:  9999994;}
+.z-index-max-minus-5  {z-index:  9999995;}
+.z-index-max-minus-4  {z-index:  9999996;}
+.z-index-max-minus-3  {z-index:  9999997;}
+.z-index-max-minus-2  {z-index:  9999998;}
+.z-index-max-minus-1  {z-index:  9999999;}
+.z-index-max          {z-index: 10000000;}
```

Comment:
WTF. Can we please stop to reindent the whole fucking code ? It makes checking the diff impossible.